sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -368,6 +349,7 @@ vpn_subscriptions AS (
 vpn_subscriptions_with_end_date AS (
   SELECT
     *,
+    MIN(subscription_start_date) OVER (PARTITION BY customer_id) AS customer_start_date,
```

Comment:
`customer_start_date` moved here from the `subscription_platform.stripe_subscriptions` view because it shouldn't be calculated across all products.